pulseapi/entries/serializers/news.py

```diff
@@ -2,6 +2,7 @@
 
 
 class NewsEntrySerializer(EntrySerializerWithCreators):
+    help_types = None
```

Comment:
as of DRF 3.7 fields may no longer be "active _and_ in the exclude list" so if they're inherited as active, they need to be explicitly set to `None` if they are to be excluded.